packages/shared/components/atoms/KeyValueBlock.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="typescript">
+    import { FontWeightText } from 'shared/components/Text.svelte'
```

Comment:
It would be nice to start creating a library specifically for the UI, then we could just import `FontWeightText` from there. There's some functions in `shared/lib/utils` at the moment that could also be moved, plus I'm sure there's some other logic elsewhere that really only applies to UI. Will give this a think :thinking:

Reply:
Yeah, this was part of another PR for the text component, but because its the props for this component I kept it there. Problem is even typing exported props in svelte doesn't give you nice to use prefills in the IDE, so I thought I would expose the enum so we don't need to worry about typing it in.